L1Trigger/L1TTwinMux/src/AlignTrackSegments.cc

```diff
@@ -28,42 +28,53 @@ void AlignTrackSegments::run( const edm::EventSetup& c) {
 
   std::vector<L1MuDTChambPhDigi> l1ttma_out;
 
-  L1MuDTChambPhDigi const* ts1=0;
-  L1MuDTChambPhDigi const* ts2=0;
+ // L1MuDTChambPhDigi const* ts1=0;
+//  L1MuDTChambPhDigi const* ts1=0;
+ // L1MuDTChambPhDigi const* ts2=0;
+//  std::auto_ptr<const L1MuDTChambPhDigi> ts1(new L1MuDTChambPhDigi);
+ // std::unique_ptr<L1MuDTChambPhDigi> ts2(new L1MuDTChambPhDigi);
 
+  auto ts1=std::make_shared<L1MuDTChambPhDigi>(0,0,0,0,0,0,7,0,0,0);
```

Comment:
As far as I can see ts1 and ts2 are only needed in the for loop, so no need to create them here.